src/components/Scheme/SchemeContainer.tsx

```diff
@@ -207,7 +207,7 @@ class SchemeContainer extends React.Component<IProps, IState> {
                         href="#!"
                         onClick={this.handleEditPlugin}
                       >
-                    Edit Plugin
+                        Edit Plugin
```

Comment:
why adding these spaces ?

Reply:
that is a mystery.  I was able to restore them and neither lint nor my editor is complaining.